PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### Your checklist for this pull request
+🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
+
+- [ ] Ensure all commits include 
```

Comment:
include what?

Reply:
It should have been the DCO sign-off, but could be rendered moot by your other comments.

Reply:
@tarcinil Fair enough. Ping me or @rvodden once you have it rebased and we'll give it another look.

Reply:
@perlun My branch has been rebased and comments addressed.